src/utils/checkEmptyValues.js

```diff
@@ -0,0 +1,11 @@
+function checkEmptyValues() {
+  for (let i = 0; i < arguments.length; i++) {
+    if (!arguments[i]) {
+      throwError(400, "KEY_ERROR");
+    }
+  }
+}
+
```

Comment:
```suggestion
function checkEmptyValues(arguments) {
  for (let i = 0; i < arguments.length; i++) {
    if (!arguments[i]) {
      throwError(400, "KEY_ERROR");
    }
  }
}

```

1. 함수 선언 시 아무 매개변수가 없는데, 이 함수 잘 동작하나요???
2. checkEmptyValues에 들어오는 arguments를 배열로 받으시는 함수라면, 함수를 호출할 때에도 배열로 인자를 넘겨주셔야 합니다. 사용되는 곳을 보니, 함수 인자 자리에 배열이 없는 것 같은데 확인 부탁드립니다!

Reply:
잘 동작합니다.
arguments는 자바스크립트 함수에 전달되는 모든 arguments를 array로 불러오는 값입니다.
https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Functions/arguments

함수에 인자로 전해주는 것이 아니어도 함수 내부에서 호출 가능합니다.

<img width="529" alt="image" src="https://github.com/wecode-bootcamp-korea/49-2nd-5Sherlock-backend/assets/52498558/7b84304f-62f0-4588-ad07-f723eda3ff21">